lib/storage/storage.js

```diff
@@ -46,15 +50,29 @@ class StorageBase {
             .replace(/=+$/, '');
     }
 
+    static isCleanText(text) {
+        const lowercased = text.toLowerCase();
+        return profanities.some(badWord => lowercased.includes(badWord));
+    }
+
     handler(req, res) {
         let config = null;
+        // Get the desired config, which we'll first check for profanities in its hash
         if (req.body.config) {
-            config = JSON.stringify(req.body.config);
+            config = req.body.config;
         } else if (req.body.sessions) {
-            config = JSON.stringify(req.body);
+            config = req.body;
         }
-
-        const configHash = StorageBase.safe64Encoded(hash(config, FILE_HASH_VERSION));
+        config.cleanings = 0;
```

Comment:
Maybe call this something else. In security this kind of value whose presence is purely there to "mix up" things is called a `nonce`.

Reply:
In terms of testing:
* we could _not_ add the `nonce` as a first attempt, and only put it in if the first time around we spot a profanity.
* then we extract this behaviour into a separate testable function
* Then we could use the real-world example from twitter's `config` as a test case